### DISCUSSION TITLE
TikTok无法连接网络

### DISCUSSION THREAD

**Discussion by ungluee** [2dust/v2rayNG#3290] [General]
新版本TikTok连接不上网络，其他app正常。其他代理工具也能正常连接。

**Comment by 582583241** [comment]
不是新版本连不上，是一直都连不上！

**Reply by lys52** [reply]
退到1.18.19以下的才能用tiktok

**Reply by V4444** [reply]
为啥呢？好像的确有这问题，相同节点。我旧手机，安装的旧版本的确就能用，新手机安装的新版本就不能用

**Comment by lys52** [comment]
对，我也是要来反应不能上tiktok

**Comment by V4444** [comment]
经过测试，使用相同节点。安装不同版本，的确是V2RAYNG有问题。使用1.8.19就可以观看，使用1.9.x就不行。困惑了我好久，一直以为是其他原因
